docs/data-sources/ddm_flavors.md

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Distributed Database Middleware (DDM)"
+---
+
+# huaweicloud_ddm_flavors
+
+Use this data source to get the list of DDM flavors.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_ddm_engines" test {
+  version = "3.0.8.5"
+}
+
+data "huaweicloud_ddm_flavors" test {
+  engine_id  = data.huaweicloud_ddm_engine.test.engines[0].id
+  group_type = "X86"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `engine_id` - (Required, String) Specifies the ID of a engine.
+
+* `group_type` - (Optional, String) Specifies the compute resource architecture type. The options are **X86** and **ARM**.
+
+* `type_code` - (Optional, String) Specifies the resource type code.
```

Comment:
it seems that the value is fixed to "hws.resource.type.ddm", I think you can delete it.

Reply:
done